beetsplug/export.py

```diff
@@ -40,6 +42,9 @@ class ExportPlugin(BeetsPlugin):
 
     def __init__(self):
         super(ExportPlugin, self).__init__()
+        # Used when testing export plugin
+        self.run_results = None
```

Comment:
Hmm; I don't think I see where this is actually used?

Reply:
I was going to use that class field as a way of checking the output during the testing of the export plugin but I ended up going another route so I will go ahead and remove it.